### PR TITLE
various ui changes for file upload

### PR DESCRIPTION
**Pull request by LBHCallumM**
### Description of change

- Add description to file upload on mobile close work order form
- Various other UI changes

### CODE REVIEW

File: src/components/WorkOrder/Photos/PhotoViewList.js [hunk 1]
```diff
@@ -75,7 +72,7 @@ const UpdateDescriptionForm = ({ description, onSubmit, fileGroupId }) => {
             name="description"
             label="Description"
             defaultValue={description ?? ''}
-            placeholder="Add a description"
+            placeholder="Add a description..."
```
Comment (humulla) [inline src/components/WorkOrder/Photos/PhotoViewList.js:75]: is it necessary to have the dots at the end?

File: src/components/WorkOrder/Photos/UploadPhotosForm.js [hunk 2]
```diff
@@ -27,50 +27,61 @@ const UploadPhotosForm = ({ workOrderReference, onSuccess }) => {
...
-                <div style={{ marginLeft: '15px', marginTop: 0 }}>
-                  {loadingStatus}
-                </div>
+          {files.length > 0 && (
+            <div className="govuk-grid-row">
+              <div className="govuk-grid-column-one-half-from-desktop">
+                <TextArea
+                  name="description"
+                  label="Description"
+                  placeholder="Write a description..."
```
Comment (humulla) [inline src/components/WorkOrder/Photos/UploadPhotosForm.js:51]: Same here, do we need the dots at the end?

File: src/components/WorkOrders/MobileWorkingCloseWorkOrderForm.js [hunk 3]
```diff
@@ -92,6 +92,17 @@ const MobileWorkingCloseWorkOrderForm = ({ onSubmit, isLoading }) => {
                     },
                   })}
                 />
+
+                {files.length > 0 && (
+                  <div style={{ marginBottom: '60px', marginTop: '15px' }}>
+                    <TextArea
+                      name="description"
+                      label="Photo description"
+                      placeholder="Write a description..."
```
Comment (humulla) [inline src/components/WorkOrders/MobileWorkingCloseWorkOrderForm.js:101]: same as above

File: src/components/WorkOrder/Photos/PhotoViewList.js (hunk 1 above)
Comment (LBHCallumM) [reply src/components/WorkOrder/Photos/PhotoViewList.js:75]: Just following the design

**Review by adamtry — APPROVED** [review]

**Pull request merged**